projects/addon-doc/src/components/example/example.component.ts

```diff
@@ -57,7 +57,7 @@ export class TuiDocExampleComponent {
 
     readonly processor$: Observable<Record<string, string>> = this.rawLoader$$.pipe(
         switchMap(rawLoadRecord),
-        map(this.processContent),
+        map((value: Record<string, string>) => this.processContent(value)),
```

Comment:
Unnecessary type

Reply:
but then the rule breaks

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/12021443/162406719-52830d89-1df6-4596-846e-88e40f063531.png">

Reply:
I mean the type is unnecessary, I'm ok with following the rule :)

Reply:
fixed